libbeat/outputs/shipper/docs/shipper.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[shipper-output]]
```

Comment:
I don't think we want this in the published beats documentation. Definitely not until the shipper is actually released.

Will this be picked up as part of the docs build by default?

Reply:
Maybe there is a way to exclude it but still have it in here so we can maintain it.

I don't want to document an interface that is experimental and completely untested at this point. 

The intent is to only use this with the agent (which provisions it automatically) so I'm not even sure we need to provide user documentation, only developer documentation.

Reply:
Hmm, I don't really see the point in publishing this unless we expect users to do something with it. To me it is something extra to maintain. The warning helps but it is like publishing documentation with a warning not to read or believe it :p

The beats documentation is quite widely read. I'm not opposed to publishing something there (especially the output section that everyone needs to read to use beats) but we should explain what this is in a lot more detail. There should be links to the shipper repository, probably I/we should add more detail to the README there, etc. We haven't publicly communicated or explained the shipper much, besides creating the repository. 

I'm not opposed to putting the output documentation in eventually. I would just prefer to do it once this is more stable and the rest of the shipper architecture is better documented in public.